### PR TITLE
Pegasus pruning - /educate subdirectory - Summer 2022

### PR DESCRIPTION
**Pull request by kelbyhawn**
Remove old, irrelevant, and draft pages from the `pegasus/sites.v3/code.org/public/educate/` directory which runs the [Code.org](https://code.org/) marketing site.

Relevant info: [Asana task](https://app.asana.com/0/0/1202778826366844/f) • [Site Audit Gsheet](https://docs.google.com/spreadsheets/d/1pRkNtthNeddMO_dsXRcb59BuidSezAHojjIMwfw-_PQ/edit#gid=0)

Related PRs: https://github.com/code-dot-org/code-dot-org/pull/47035 • https://github.com/code-dot-org/code-dot-org/pull/47425

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/student-council.md.erb [hunk 1]
```diff
@@ -1,60 +0,0 @@
...
-
-<style>
-  details summary {
-    cursor: pointer;
-  }
-</style>
-
-<br>
-
-<center><img src="/images/csa/student-council-2.jpg", width=100%></center>
```
Comment (davidsbailey) [inline pegasus/sites.v3/code.org/public/educate/student-council.md.erb:23]: looks like you could also consider removing pegasus/sites.v3/code.org/public/images/csa/student-council-2.jpg

**Comment by kelbyhawn** [comment]
@davidsbailey @breville hey guys, this should be the last Pegasus pruning PR (at least for awhile), and I worked w/ the curriculum and edpop teams to determine what can be removed here.

**Review by davidsbailey — APPROVED** [review]
awesome!!! looks good to me

File: pegasus/sites.v3/code.org/public/educate/student-council.md.erb (hunk 1 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/educate/student-council.md.erb:23]: Done! (might try to tackle the images folder next summer 😅)

**Review by breville — APPROVED** [review]

**Pull request merged**